Hermes.xcodeproj/project.pbxproj

```diff
@@ -652,6 +652,11 @@
 			attributes = {
 				BuildIndependentTargetsInParallel = YES;
 				LastUpgradeCheck = 0730;
+				TargetAttributes = {
+					8D1107260486CEB800E47090 = {
+						DevelopmentTeam = 8END7325T3;
```

Comment:
This is your personal key? We should likely not commit this into the general repository.